PUArallelepiped/PUN-street-Universal-Access
# Check Order Page Auto Refresh

resolve #134 

### do
1. client connect to server via web socket when in /checkOrder page
2. /checkOrder page will auto reload when order's status changed

### test
1. open two browser (customer and seller, customer should buy something in seller's store)
2. customer go http://localhost:5050/order/checkOrder, seller go http://localhost:5050/order/updateOrderStatus
3. update order's status with seller

### todo
1. maybe use redis to manage subscribers would be better
